tools/gulp/tasks/unit-test.ts

```diff
@@ -75,13 +75,15 @@ gulp.task('test', [':test:deps'], () => {
   });
 
   // Refreshes Karma's file list and schedules a test run.
-  let runTests = () => {
-    server.refreshFiles().then(() => server._injector.get('executor').schedule());
+  let runTests = (err?: Error) => {
+    if (!err) {
+      server.refreshFiles().then(() => server._injector.get('executor').schedule());
+    }
   };
 
   // Boot up the test server and run the tests whenever a new browser connects.
   server.start();
-  server.on('browser_register', runTests);
+  server.on('browser_register', () => runTests());
```

Comment:
Because this event comes from Karma and it calls the callback with the `browser_data` as first argument.